tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py

```diff
@@ -185,7 +185,9 @@ def test_backward_no_reduction_gpu_no_cudnn(self):
 
 
 @testing.parameterize(
-    {'use_cudnn': 'always'},
+    # TODO(okuta) : fix
+    # TypeError: Unsupported type <class 'mock.mock.MagicMock'>
+    # {'use_cudnn': 'always'},
```

Comment:
What causes this error?

Reply:
`mock` returns `mock.mock.MagicMock` type. So, `isinstance` is failed.

Reply:
How about:
```
            with mock.patch.object(
                    cuda.cupy.cudnn, 'activation_forward',
                    wraps=cuda.cupy.cudnn.activation_forward) as func:
```

Reply:
I will try